lxd/db/cluster/auth_groups.go

```diff
@@ -287,7 +287,7 @@ func SetAuthGroupPermissions(ctx context.Context, tx *sql.Tx, groupID int, authG
 	}
 
 	for _, permission := range authGroupPermissions {
```

Comment:
As well as this change, should we check the Permissions against the groupID and return an error before trying to execute the query?

Reply:
Ah I see its removed in next commit.